### ISSUE TITLE
Migrate from Prefect Cloud 1 --> Prefect Cloud 2 by July 15th, 2023

### ISSUE THREAD

**Issue by smcclure17** [act-now-coalition/can-scrapers#468]
Context: https://www.prefect.io/guide/blog/freezing-legacy-prefect-cloud-1-accounts-on-starter-and-standard-plans/

We use Prefect Cloud 1 at the moment. I believe this is a separate task from updating our workflows from Prefect 1 to Prefect 2, which requires significant code changes

**Comment by smcclure17** [comment]
Need to make a go/no-go decision on whether or not to continue running scrapers? Probably contingent on data landscape after [May 11th](https://oig.hhs.gov/coronavirus/covid-flex-expiration.asp)
